app/src/lib/databases/repositories-database.ts

```diff
@@ -23,6 +23,11 @@ export interface IDatabaseGitHubRepository {
   readonly lastPruneDate: number | null
 }
 
+export interface IDatabaseProtectedBranch {
+  readonly repoId: number
+  readonly name: string
```

Comment:
I'm fine with the name of this property but it could use a sprinkle of documentation to make it easy to understand that it's a branch name it's referring to (and also whether than branch name is fully qualified or not).

Reply:
Added in 3581f36e87387ae4fa8beba20ab9a0b0d90c630c